### DISCUSSION TITLE
course/starbook/appendix

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#66] [General]
# course/starbook/appendix

本书提供了一系列开发功能的优化建议，包括中间件的使用和自定义错误信息，将Jwtkey纳入配置文件，实施JWT黑白名单机制以及在入库数据时进行验证。通过这些方向，读者可以在加深对GoFrame和Golang的理解的同时，提升编程技能，并在业务优化中获得更多经验。

https://goframe.org/course/starbook/appendix

**Comment by qvoo** [comment]
goooooooooooooooooooooooooooooooooooooooooood!

**Comment by 1378496782** [comment]
# Thanks very much for your awesome tutorial!!!
